lib/core/jhipster/field_types.js

```diff
@@ -97,7 +99,9 @@ const CassandraValidations = {
   Boolean: new Set([REQUIRED, UNIQUE]),
   Date: new Set([REQUIRED, UNIQUE]),
   UUID: new Set([REQUIRED, UNIQUE]),
-  Instant: new Set([REQUIRED, UNIQUE])
+  Instant: new Set([REQUIRED, UNIQUE]),
+  BYTE_BUFFER: new Set([REQUIRED, UNIQUE, MINBYTES, MAXBYTES]),
+  ZONED_DATE_TIME: new Set([REQUIRED, UNIQUE])
```

Comment:
These two attributes need be like the others, like `ByteBuffer`

Reply:
Ah, missed that. Done 😄